### ISSUE TITLE
Pause movie search on indexer failures 

### ISSUE THREAD

**Issue by Bastien42** [Radarr/Radarr#5170]
**Issue**

A problem i've had was that i've an extensive library 60k+ and when i was performing manual search on my only indexer well indexer stability is not alway perfect and it would crash from time to time issue being that in this case the movie search will skip movie really fast as 0 indexer is active 

**Solution**

As a solution i thought of something like this that when your indexer fails the manual movie search is paused until the so said indexer is back online and of course while it's pause radarr should refresh the status of the indexer 

**PS**

i've no idea if i'm the only one having that but i figured that may not be too complicated to have this feature and would become really practical as i would just launch a manual research over my 60k movies and even if my indexer crashes from time to time at the end i would know that every movie have been searched individually and none were skipped due to indexer failures

**Comment by austinwbest** [comment]
I can see a benefit to this on a large library but im not sure if each movie is updated as it cycles with a last updated time & if not then it wouldn't know where to pick back up. Q or ta would know that one better than would.

**Comment by Bastien42** [comment]
yeah i don't know how that would have to be implemented in radarr or if it even possible nut yeah that would a great feature for an already awesome project that i hope will be long lasting 👍

**Comment by Bastien42** [comment]
adding some info to the feature request the tracker i'm using is obviously timing me out due no numerous request which is ok only if such a feature is available to not let 0 indexer search for movie as it jut crawl through for nothing
